core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/wait.yml

```diff
@@ -15,7 +15,7 @@
   shell: kubectl get nodes -o json
   register: output
   until: output.stdout|from_json|json_query("items[*].status.conditions[?(@.type=='Ready')].status[]")|unique == ["True"]
-  retries: 600
+  retries: 600 # 20min
```

Comment:
Won't be those comments misleading if someone will forget to change number of minutes after changing retries?

Reply:
Comments are for me, someone can remove or update them when making changes in this area. We have code reviews to not forget :)